config/constants.yml

```diff
@@ -39,9 +39,19 @@ localizedUrls:
     aboutPrivacy: https://about.qwant.com/legal/privacy
     aboutMapsToS: https://about.qwant.com/legal/terms-of-service/qwant-maps/
     contributing: https://github.com/Qwant/qwantmaps/blob/master/contributing.md
+    helpGeolocation: https://help.qwant.com/en/docs/maps/activation-of-geolocation-after-denied-access/
```

Comment:
It does not depend on this PR, but images contained into these links are missing for me (on Firefox) :sweat_smile:

Reply:
Same for me, they come from googleusercontent.com so there is a CSP error :disappointed: